docs/package.json

```diff
@@ -92,19 +92,19 @@
     "lz-string": "^1.4.4",
     "markdown-to-jsx": "^7.1.7",
     "material-ui-popup-state": "^2.0.1",
-    "next": "12.1.5",
+    "next": "12.2.0",
```

Comment:
I could pinpoint this change to an ocean of warnings in the docs 😁

<img width="640" alt="Screenshot 2022-07-15 at 23 58 03" src="https://user-images.githubusercontent.com/3165635/179316506-9b04d9f4-6343-4140-bcc1-644e5acbd53b.png">

Reply:
@mnajdova Found another one while debugging in Netlify, the `yarn workspace docs export` now spawns a bunch of [warnings around the usage of `getInitialProps`](https://nextjs.org/docs/messages/get-initial-props-export):

```
warn  - Detected getInitialProps on page '/base/api/badge-unstyled'while running "next export". It's recommended to use getStaticPropswhich has a more correct behavior for static exporting.
Read more: https://nextjs.org/docs/messages/get-initial-props-export
```

Reply:
Another regression: the [scroll restoration](https://majido.github.io/scroll-restoration-proposal/history-based-api.html) is broken on my end:

- It works on v5.8.7 https://62c35000f3955c000870e453--material-ui-docs.netlify.app/
- but doesn't work on this PR: https://deploy-preview-33196--material-ui.netlify.app/.

 I think that it's a regression inside Next.js itself, moving from 12.1.5 to 12.2.0.

Reply:
I will try to look into these by the end of the week. If I don't find time I will ask someone in the team to look into them.